playground/package.json

```diff
@@ -23,7 +23,7 @@
     "@typescript-eslint/eslint-plugin": "^8.2.0",
     "@typescript-eslint/parser": "^8.2.0",
     "@vitejs/plugin-react-swc": "^3.7.0",
-    "eslint": "^9.9.0",
+    "eslint": "^8.57.0",
```

Comment:
Question: why did you have to revert back?

Reply:
As stated in the PR description, another dependency doesn't support version 9 of eslint and the build would fail as a result.

Reply:
Sorry my brain thought I had written about the eslint issue in the PR description but I did that in the linked issue.

Reply:
I had read the linked issue as well. No worries.

But the added flag to `npm i` fixes the dep issue no?

Where in the build does it fail with the new eslint version?

Reply:
You are referring to the react hooks plugin, right?

Reply:
Discussed on Discord. Added `--legacy-peer-dep` to `make deps:playground` keeping `eslint` at version 9.9.